example/ios/Demo.xcodeproj/project.pbxproj

```diff
@@ -366,7 +366,8 @@
 				ORGANIZATIONNAME = Splendo;
 				TargetAttributes = {
 					74555136E1B2BB715E50195E = {
-						ProvisioningStyle = Manual;
+						DevelopmentTeam = 2732HQPVJP;
```

Comment:
Not a good idea to share team id in public repo

Reply:
Removed